src/actionTypes/updateUserReg.js

```diff
@@ -0,0 +1,5 @@
+export const completeRegistration = {
+  COMPLETE_REG_REQUEST: 'COMPLETE_REG_REQUEST',
+  COMPLETE_REG_SUCCESS: 'COMPLETE_REG_SUCCESS',
+  COMPLETE_REG_FAILURE: 'COMPLETE_REG_FAILURE',
+}
```

Comment:
#### Nitpick
Add a new line here.

Reply:
done... I wasn't the one that work on those files